libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -239,8 +239,15 @@ const AP_Param::GroupInfo SRV_Channels::var_info[] = {
     // @Bitmask: 0:Servo 1, 1:Servo 2, 2:Servo 3, 3:Servo 4, 4:Servo 5, 5:Servo 6, 6:Servo 7, 7:Servo 8, 8:Servo 9, 9:Servo 10, 10:Servo 11, 11:Servo 12, 12:Servo 13, 13:Servo 14, 14:Servo 15, 15:Servo 16, 16:Servo 17, 17:Servo 18, 18:Servo 19, 19:Servo 20, 20:Servo 21, 21:Servo 22, 22:Servo 23, 23:Servo 24, 24:Servo 25, 25:Servo 26, 26:Servo 27, 27:Servo 28, 28:Servo 29, 29:Servo 30, 30:Servo 31, 31:Servo 32
     // @User: Advanced
     // @RebootRequired: True
-    AP_GROUPINFO("_GPIO_MASK",  26, SRV_Channels, gpio_mask, 0),
+    AP_GROUPINFO("_GPIO_MASK",  26, SRV_Channels, gpio_mask, 0),  
 
+    // @Param: _PASTHRU_FS
+    // @DisplayName: Servo Passthru FS mask
+    // @Description: This sets a bitmask of scaled passthru output function which will be set to their trim value during failsafe instead of holding their last valid value.
```

Comment:
nitpick: no need for the "This sets a" part of the description because all parameters are for setting something.

Reply:
We should specify that this applies to RC failsafe.  So "during RC failsafe"

Reply:
done...and fixed all other params with that "this sets"